catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -949,7 +976,10 @@ private ExpressionValues getExpressions(Expression expression1, Expression expre
         String propertyName;
         Object literal;
 
-        if (expression1 instanceof PropertyName && expression2 instanceof Literal) {
+        if (expression1 instanceof Function && expression2 instanceof Literal) {
+            return new ExpressionValues((ProximityArguments) expression1.accept(this, delegate),
```

Comment:
How do we know if it is a function we should be looking for ProximityArguments?  Does this mess up future extensions?